storage/range_test.go

```diff
@@ -160,6 +160,7 @@ func (tc *testContext) Start(t testing.TB) {
 			}
 		}
 		tc.store.ctx.DB = client.NewKV(nil, &testSender{store: tc.store})
+		tc.store.rangeGCQueue.db = tc.store.ctx.DB
```

Comment:
Thanks for the suggestion, @tschottdorf ! The trick worked. I set `ctx.DB` to `client.NewKV(nil, sender)` instead of `tc.store.ctx.DB`.

`make testrace` was flaky, but I think that's #1010. Will rebase and run the test again once it is fixed.